CMakeLists.txt

```diff
@@ -518,6 +520,40 @@ foreach(target IN LISTS targets)
     link_common_libraries(${target})
 endforeach()
 
+if (fuzzer_conditions)
```

Comment:
I'd like to keep this variable. If we add other fuzzers I think we want to control which fuzzers we are building. I don't mind the slightly clunkier build step for this because it's an uncommon build target.